observability/tools/OpenTelemetry.md

```diff
@@ -0,0 +1,32 @@
+# Open Telemetry
+
+Open Telemetry is a set of APIs, SDKs and instrumentation which can be used for collection, processing and orchestration of telemetry data like traces metrics and logs. It supports multiple languages(Java, dotnet, python, javascript, golang, erlang etc.). Open telemetry follows a vendor agnostic and standards based approach for collection and management of telemetry data. Important point to note is that OpenTelemetry does not have its own backend, all telemetry collected by OpenTelemetry Collector must be sent to a backend like Prometheus etc. Open telemetry is also the 2nd most active CNCF project after Kubernetes.
+
+## Collector
+
+OpenTelemetry Collector performs telemetry collection, processing and export. It is mostly run as an agent or a sidecar with the application or as a daemon if installed directly on host. As the collector is standards based and supports many integrations, there is a possibility that many tool specific agents for metric, traces and logs can be replaced by a single OpenTelemetry collector.
+The collector can also be installed as standalone service to receive telemetry data from multiple applications.
+
+## Instrumentation Libraries
+
+A library that enables observability for another library is called an instrumentation library. OpenTelemetry libraries are language specific, currently there is good support for Java, Python, Javascript, dotnet and golang. Support for automatic instrumentation is available for some libraries which make using OpenTelemetry easy and trivial. In case automatic instrumentation is not available, manual instrumentation can be configured by using the OpenTelemetry SDK.
+
+## Integration of OpenTelemetry
+
+OpenTelemetry can be used to collect, process and export data into multiple backends, some popular integrations supported with OpenTelemetry are:
+
+1. Zipkin
+2. Prometheus.
+3. Jaeger
+4. New Relic.
+5. Azure Monitor
+
+## Why use OpenTelemetry
+
+The main reason to use OpenTelemetry is that it offers tracing, metrics and logging telemetry through a single agent which supports multiple integrations. Seperate agents for tracing, logging and metrics will not be needed to be setup if OpenTelemetry is used.
+
```

Comment:
I believe a section on the current status of the OT project would be helpful here. 
Notes on this tool if can be recommended for production ready scenarios, what to watch for, what are the benefits of this tool would be interesting. 
I believe adding this kind of information would be helpful especially for dev crews. @khaines  @MagdaPaj  - adding you for insights into what your thoughts are.

Reply:
+1 on detailed guidance. This is a great intro and summary of OpenTel, but adding some content to fill out the areas @NarmathaBala highlights would be ideal.

Reply:
I think it looks really good now. I would add somewhere maybe this link about project status: https://opentelemetry.io/project-status/ which clearly states what's the current status.

And one more suggestion. One of the problems that we noticed when we were evaluating OT was incompatible versions between Open Telemetry Library and the exporter that we wanted to use (see it [here](https://github.com/Azure/azure-sdk-for-net/issues/17253)). Once everything will be GA, then this should be solve. But I think it will take time until this happens for all languages and exporters. So maybe we could briefly mention to pay attention to it. Because if someone has to use specific exporter, and then Open Telemetry is not compatible with it, then it doesn't make a lot of sense to invest time in it.

Reply:
I really like this added section @lakshaykaushik @MagdaPaj